front/app/translations/admin/en.json

```diff
@@ -32,10 +32,11 @@
   "app.components.Admin.Campaigns.deliveryStatus_accepted": "Accepted",
   "app.components.Admin.Campaigns.deliveryStatus_bounced": "Bounced",
   "app.components.Admin.Campaigns.deliveryStatus_clicked": "Clicked",
-  "app.components.Admin.Campaigns.deliveryStatus_clickedTooltip": "When you added one or more links to your email, the number of users who clicked a link will be shown here.",
+  "app.components.Admin.Campaigns.deliveryStatus_clickedTooltip": "This shows how many recipients clicked a link in the email. Please note that some security systems may follow links automatically to scan them, which can result in false clicks.",
```

Comment:
Not sure what has happened here as you have not changed this in the messages file. Should probably add this text to the messages file and change the ID to `app.components.Admin.Campaigns.deliveryStatus_clickedTooltip2`

Reply:
Alright, will do that. I thought that existing copy should be changed that way, but it probably works differently for the frontend.